Tools/clinic/libclinic/converter.py

```diff
@@ -531,3 +531,22 @@ def add_include(self, name: str, reason: str,
 # these callables follow the same rules as those for "converters" above.
 # note however that they will never be called with keyword-only parameters.
 legacy_converters: ConverterDict = {}
+
+
+def add_legacy_c_converter(
+        format_unit: str,
+        **kwargs: Any
```

Comment:
Could you dedent these one level according to typing style conventions? I probably introduced this weird style, but this is a good opportunity to amend it.